website/docs/r/cbr_rule.html.markdown

```diff
@@ -14,75 +14,132 @@ Provides a resource for cbr_rule. This allows cbr_rule to be created, updated an
+        * `attributes` - (Required, List) The resource attributes.
+            * Constraints: The minimum length is `1` item.
+              Nested scheme for **attributes**:
+                * `name` - (Required, String) The attribute name.
+                    * Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `^[a-zA-Z0-9]+$`.
+                * `operator` - (Optional, String) The attribute operator.
+                    * Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `^[a-zA-Z0-9]+$`.
+                * `value` - (Required, String) The attribute value.
+                    * Constraints: The maximum length is `1000` characters. The minimum length is `1` character. The value must match regular expression `^[\S\s]+$`.
+        * `tags` - (Optional, List) The optional resource tags.
+            * Constraints: The maximum length is `10` items. The minimum length is `1` item.
+              Nested scheme for **tags**:
+                * `name` - (Required, String) The tag attribute name.
+                    * Constraints: The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `^[a-zA-Z0-9 _.-]+$`.
+                * `operator` - (Optional, String) The attribute operator.
+                    * Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `^[a-zA-Z0-9]+$`.
+                * `value` - (Required, String) The tag attribute value.
+                    * Constraints: The maximum length is `1000` characters. The minimum length is `1` character. The value must match regular expression `^[a-zA-Z0-9 _*?.-]+$`.
 ## Attribute Reference
 
 In addition to all argument references listed, you can access the following attribute references after your resource is created.
 
 * `id` - The unique identifier of the cbr_rule.
-* `created_at` - (String) The time the resource was created.
-* `created_by_id` - (String) IAM ID of the user or service which created the resource.
-* `crn` - (String) The rule CRN.
-* `href` - (String) The href link to the resource.
-* `last_modified_at` - (String) The last time the resource was modified.
-* `last_modified_by_id` - (String) IAM ID of the user or service which modified the resource.
+* `created_at` - (Required, String) The time the resource was created.
+* `created_by_id` - (Required, String) IAM ID of the user or service which created the resource.
+* `crn` - (Required, String) The rule CRN.
+* `href` - (Required, String) The href link to the resource.
+* `last_modified_at` - (Required, String) The last time the resource was modified.
+* `last_modified_by_id` - (Required, String) IAM ID of the user or service which modified the resource.
 
 * `version` - Version of the cbr_rule.
 
+## Provider Configuration
```

Comment:
Remove this section from docs ..we rasied an issue with generator team

Reply:
removed